chroma-manager/tests/integration/shared_storage_configuration/test_updates.py

```diff
@@ -28,18 +27,21 @@ def test_upgrade_alerting(self):
 
         self.assertNotEqual(len(original_packages), 0)
 
-        self.assertNoAlerts(host['resource_uri'], of_type='UpdatesAvailableAlert')
+        if config.get('simulator'):
+            # the simulator only does a package scan on initial (fake) plug-in update,
+            # therefore we have to do it manually to avoid race conditions failing this test
```

Comment:
Is this a race we just **always** won before this patch or is there some other difference that is requiring that we force this `scan_packages()` now?